music_rnn/demo/index.ts

```diff
@@ -155,15 +155,16 @@ async function runMelodyRnn() {
 }
 
 async function runDrumsRnn() {
-  const drumsRnn = new MusicRNN(DRUMS_CHECKPOINT, null, 32);
+  const drumsRnn = new MusicRNN(
+      DRUMS_CHECKPOINT, null, 32,
+      magenta.controls.controlSignalFromSpec(
```

Comment:
Can we set this up to automatically load from the checkpoint or is that a later PR?